### ISSUE TITLE
[Feature Request] Allow bucket name to be specified in Backblaze B2 backend setup for single bucket keys.

### ISSUE THREAD

**Issue by hector-hyrivera** [mickael-kerjean/filestash#781]
If you create an application key that only has access to a single bucket, it will fail in the Init. This is due to B2 declining all "list_buckets" request if the key is limited to one bucket and the bucket name is not in the request. (See https://b2-sdk-python.readthedocs.io/en/master/api/api.html#b2sdk.v2.B2Api.list_buckets)

I believe it would be beneficial to add the ability to pass in the bucket name so we can limit which buckets are exposed to the application key we generate for Filestash.

**Comment by mickael-kerjean** [comment]
That's what the path is about, just put: /bucketname/ and you will be chrooted onto that bucket

**Comment by adriantunez** [comment]
@mickael-kerjean I'm working with different buckets using different s3 buckets using the path suggested here. Is there any way to show in which bucket we're in?

<img width="1109" height="380" alt="Image" src="https://github.com/user-attachments/assets/e1dd8653-eec6-4aec-84f7-209e0c4821b0" />

Thx in advanced!

**Comment by mickael-kerjean** [comment]
Yes, it’s possible but it requires some custom connector development:

1. Using a virtual FS layer where you manually mount different paths

2. Extend the S3 connector with a plugin to add a custom field showing the current bucket

I have done similar things for customers that need custom behavior but none of this is general enough to be integrated onto the core
